eqs/src/configuration.rs

```diff
@@ -27,9 +27,35 @@ pub struct EQSOptions {
     #[structopt(long = "store_path", short = "s", default_value = "")]
     pub store_path: String,
 
+    /// URL for Ethers HTTP Provider
+    #[structopt(
+        long = "rpc_url",
+        env = "RPC_URL",
+        default_value = "http://localhost:8545"
+    )]
+    pub rpc_url: String,
+
+    /// Mnemonic for Ethers wallet
```

Comment:
@nyospe What will the EQS use the wallet for?

Reply:
It shouldn't, but the EthConnection setup needs it.

Reply:
Ok. I will try improve that.